MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs

```diff
@@ -178,7 +182,14 @@ private void WelcomeScreenDisplay(SessionBase session)
         {
             EchoToClient(session, new byte[] { 0x1B, 0x5B, 0x32, 0x4A });
             EchoToClient(session, new byte[] { 0x1B, 0x5B, 0x48 });
-            EchoToClient(session, _resourceManager.GetResource("MBBSEmu.Assets.login.ans").ToArray());
+            string LoginFileName = _configuration["Login.File"];
+            //Check if file is specified in appsettings.json and if it exists, else display default
+            if (File.Exists(LoginFileName)) {
+                byte[] LoginFileContents = File.ReadAllBytes(LoginFileName);
+                EchoToClient(session, LoginFileContents.ToArray());
```

Comment:
You can inline `File.ReadAllBytes()` here not having to declare the variable since it's only used the one time